Library/Formula/osquery.rb

```diff
@@ -3,9 +3,7 @@ class Osquery < Formula
   homepage "https://osquery.io"
   # pull from git tag to get submodules
   url "https://github.com/facebook/osquery.git",
-      :tag => "1.5.3",
-      :revision => "2a71162b0cfb040a3606538857f512ea8894b9e7"
-  revision 2
+    :tag => "1.6.0"
```

Comment:
Please keep the `:revision` commit reference. It acts as both a checksum and a way we can be sure the upstream tag doesn't silently wander off.